docs/SplunkOperatorUpgrade.md

```diff
@@ -144,7 +147,9 @@ This is an example of the process followed by the Splunk Operator if the operato
 ​
 1. A new Splunk Operator pod will be created, and the existing operator pod will be terminated.
 2. Any existing License Manager, Search Head, Deployer, ClusterManager, Standalone pods will be terminated to be redeployed with the upgraded spec.
```

Comment:
Do we need to mention the order here properly?

Reply:
Updated the steps